src/sysl/importers/import_openapi.py

```diff
@@ -0,0 +1,520 @@
+            if 'parameters' in api:
+                apiParams = api['parameters']
+                del api['parameters']
+
+            for (i, (method, body)) in enumerate(sorted(api.iteritems(), key=lambda t: METHOD_ORDER[t[0]])):
+                bodyContent = OrderedDict()
+                bodyParams = body['parameters'] if 'parameters' in body else []
+                pathParams, headerParams, queryParams = self.mergeParams(oaSpec, apiParams, bodyParams)
+                print(pathParams)
+                w(u'\n{}:', self.translate_path_template_params(path, pathParams))
+                with w.indent():
+                    header = self.getHeaders(headerParams)
+                    jsonBody = []
+                    if body.get('requestBody', None):
+                        if body['requestBody'].get('content', None):
+                            if body['requestBody']['content'].get('application/json', None):
+                                jsonBody.append(body['requestBody']['content']['application/json'])
+                    methodBody = self.getBody(jsonBody)
+                    if len(header) > 0 and len(methodBody) > 0:
+                        paramStr = ' ({})'.format(methodBody + ', ' + header)
+                    elif len(header) == 0 and len(methodBody) == 0:
+                        paramStr = ''
+                    else:
+                        paramStr = ' ({})'.format(methodBody + header)
+
+                    w(u'{}{}{}{}:',
+                        method.upper(),
+                        paramStr,
+                        ' ?' if queryParams else '',
+                        '&'.join(
+                            '{}={}{}'.format(p['name'], TYPE_MAP[p['schema']['type']], '' if p.get('required', None) else '?')
+                            for p in queryParams
+                        ))
+                    with w.indent():
+                        for line in textwrap.wrap(
+                                body.get('description', 'No description.').strip(), 64):
+                            w(u'| {}', line)
+
+                        # Backwards compat: support integer response keys.
+                        responses = {str(k): v for (k, v) in body['responses'].iteritems()}
+
+                        # Valid keys of responses can be:
+                        # - http status codes (I believe they must be string values, NOT integers, but spec is unclear).
+                        # - "default"
+                        # - any string matching the pattern "^x-(.*)$""
+                        # ref: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#responses-object
+
+                        includeResponses(responses)
+
+    def writeDefs(self, oaSpec, w):
+        w()
+        w('#' + '-' * 75)
+        w('# definitions')
+
+        if not oaSpec.get('components', None):
+            return
+        if not oaSpec['components'].get('schemas', None):
+            return
```

Comment:
```python
schemas = oaSpec.get('components', {}).get('schemas', None)
if not schemas:
    return
```

Reply:
Changed and substituted occurrences of `oaSpec['components']['schemas']` for `schemas`.